def do_exception(name, keyword):
    print("Error of crawling " + keyword + " in " + name)
